src/pages/trade-types/options/digital-options/up-and-down/index.tsx

```diff
@@ -1,9 +1,19 @@
 import React from 'react'
+import { Helmet } from 'react-helmet'
 import { WithIntl } from 'components/localization'
 import UpAndDownDigitalOptionsPage from 'features/pages/options/digital/up-down'
 
 const DigitalOptions = () => {
-    return <UpAndDownDigitalOptionsPage />
+    const canonicalUrl = '/trade-types/options/digital-options/up-and-down'
```

Comment:
i recommend to use `window.location.origin` before this, as there's no certainty of how search engines crawlers handle relative path.